src/orders/components/OrderPayment/OrderPayment.tsx

```diff
@@ -143,15 +143,24 @@ const OrderPayment: React.FC<OrderPaymentProps> = props => {
                 />
               </td>
               <td>
-                {maybe(() => order.shippingMethodName) === undefined &&
-                maybe(() => order.shippingPrice) === undefined ? (
+                {order?.shippingMethodName === undefined &&
+                order?.shippingPrice === undefined &&
+                order?.collectionPointName === undefined ? (
                   <Skeleton />
                 ) : order.shippingMethodName === null ? (
-                  intl.formatMessage({
-                    defaultMessage: "does not apply",
-                    description: "order does not require shipping",
-                    id: "orderPaymentShippingDoesNotApply"
-                  })
+                  order.collectionPointName === null ? (
+                    intl.formatMessage({
```

Comment:
all messages should be extracted to local `messages.ts` file

Reply:
Done.